docs/iris/src/whatsnew/contributions_1.8/newfeature_2014-12-19_netcdf-streaming.txt

```diff
@@ -1,4 +1,5 @@
 * NetCDF saving to all NetCDF4/HDF encodings (including the default) now uses biggus save when the cube has lazy data.
+
```

Comment:
As the 1.8 release wasn't compiled using this script, my instinct would be to delete the contributions_1.8 directory entirely, as it's not relevant.